qctoolkit/pulses/pulse_template.py

```diff
@@ -118,7 +118,7 @@ def create_program(self, *,
         if parameters is None:
             parameters = dict()
         if measurement_mapping is None:
-            measurement_mapping = dict()
+            measurement_mapping = dict() # todo (2018-08-02): should not be empty but an identity mapping; requires PT to be a MeasurementDefiner (?)
```

Comment:
`measurement_names` is an abstract property of `PulseTemplate`

Reply:
uh... done :D